Third Feedback Implementation

noordean opened the pull request:
#### What does this PR do?
- Implements third feedback
#### Description of Task to be completed?
- Implement the feedback gotten from facilitator
#### How should this be manually tested?
- Nil
#### Any background context you want to provide?
- Nil
#### What are the relevant pivotal tracker stories?
- #150353021
#### Screenshots (if appropriate)
Nill
#### Questions:
Nill

noordean committed:
chore(implementation): implement feedback

    - make use of case sensitive username for login
    - update README with wiki in how-to-contribute section
    - fix naming convention where necessary
    - put dev database credentials in env
    - improve app's responsiveness for smaller screens
    - refactor components into container and presentation

    [Delivers]

noordean committed:
bug(bug-fix): remove autogenerated files

- delete bundle folder

[Fixes]

temilaj commented on client/actions/MessageActions.js:
```diff
@@ -9,7 +11,7 @@ export default class MessageActions {
   * @static
 
-  * @param {Integer} groupID The id of the group to post message to
+  * @param {Integer} groupId The id of the group to post message to
```
is integer a datatype in Js?

noordean committed:
bug(bug-fix): fix bug in  addMember modal

- fix bugs in addMember modal
- remove autogenerated files

[Fixes]

noordean committed:
Merge branch 'staging' of https://github.com/noordean/PostIt into chore/third-feedback-implementation
update third-feedback-implementation with staging

noordean committed:
refactor(code-refactoring): implement feedback

- verify token for every request
- check for 500 error in every request
- account for 500 error on the client side

[Delivers]

noordean committed:
refactor(code-refactoring): improve test coverage

- write more controller test to increase coverage

[Fixes]

noordean committed:
bug(bug-fix): implement feedback

    - create middlewares for proper error handling

    [Fixes]

noordean committed:
bug(bug-fix): improve error handling

    - improve error handling in group controller

    [Fixes]
